src/MigrationTools.Clients.AzureDevops.Rest/Processors/Pipelines/ReleaseDefinitions.cs

```diff
@@ -10,4 +12,591 @@ public partial class ReleaseDefinitions
         [JsonProperty("value")]
         public ReleaseDefinition[] Value { get; set; }
     }
+
+    public partial class ReleaseDefinition : ReleaseBuildDefinitionAbstract
+    {
+        [JsonProperty("source")]
```

Comment:
Isnt Newtonsoft json not so strict on casing on properties, so all this attribs isnt needed where it is only relating to casing?

Reply:
I'll try that out

Reply:
You're right, it works